<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gift</class>
 <widget class="QWidget" name="Gift">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подарки</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>371</width>
     <height>561</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BackToMenu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад в меню</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FormGift">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Сформировать подарок</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите вес</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите цену</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите типы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineWeight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePrice">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTypes">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveToFile">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить подарки в файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
